integration-test/helpers/pages.js

```diff
@@ -272,6 +294,15 @@ export function loginPage (page, server, opts = {}) {
 
             expect(mockCalls.length).toBe(0)
         },
+        /**
+         * This is used mostly to avoid false negatives when we check for something _not_ happening.
+         * Basically, you check that a specific call hasn't happened but the rest of the script ran just fine.
+         * @returns {Promise<void>}
+         */
+        async assertAnyMockCallOccurred () {
+            const calls = await page.evaluate('window.__playwright.mocks.calls')
+            expect(calls.length).toBeGreaterThan(0)
+        },
```

Comment:
Again, this helps us reduce flakey tests. In one of the tests, we wanted to ensure that the script ran even though a specific call wasn't made. The previous solution checked for a specific number of calls to happen, but that made the test flakey. This checks the same thing without being strict on the number of calls executed.